samples/java-spring-shopping-cart-quickstart/pom.xml

```diff
@@ -22,15 +22,6 @@
   </properties>
 
   <dependencies>
-    <dependency>
-      <groupId>io.kalix</groupId>
-      <artifactId>kalix-spring-boot-starter</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>io.kalix</groupId>
-      <artifactId>kalix-spring-boot-starter-test</artifactId>
-      <scope>test</scope>
-    </dependency>
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-api</artifactId>
```

Comment:
I'm now wondering if even the `junit-jupiter-api` dependency should go on the the parent pom.. we depend on it from the testkit IIRC.

Reply:
yes, I think we can include it by default in the parent pom.

Reply:
Both JUnit deps should be pushed up
```xml
    <dependency>
        <groupId>org.junit.jupiter</groupId>
        <artifactId>junit-jupiter-api</artifactId>
        <scope>test</scope>
    </dependency>
    <dependency>
        <groupId>org.junit.jupiter</groupId>
        <artifactId>junit-jupiter</artifactId>
        <scope>test</scope>
    </dependency>
```

Reply:
Or would Java developers ever NOT use JUnit?

Reply:
I thought that Junit was required to compile/run the testkit but now I can run them even without those deps.. what am I missing? I remember there was a reason to add those...

Also, isn't it redundant to add `junit-jupiter` and `junit-jupiter-api` since the former is an [aggregator](https://hub.packtpub.com/junit-5-4-released-with-an-aggregate-artifact-for-reducing-your-maven-and-gradle-files/)?